alerts.yaml

```diff
@@ -161,3 +161,12 @@ serverFiles:
           annotations:
             summary: 'Parking model variance exceeds 0.10'
             description: 'Parking prediction variance for zone {{ $labels.zone }} is {{ $value }}.'
+      - name: low_disk_space
+        rules:
+        - alert: LowDiskSpace
+          expr: (kubelet_volume_stats_available_bytes / kubelet_volume_stats_capacity_bytes < 0.85) * 100
```

Comment:
This line doesn't look right to me.  I believe the expr is supposed to evaluate to a boolean value and I'm not sure what `(true) * 100` would do...